react-app/src/native/components/leagues/TeamList.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Text, Card, CardItem} from 'native-base';
+import { Table, Row } from 'react-native-table-component';
+import { StyleSheet } from 'react-native';
+import Loading from '../common/Loading';
+
+export default class Schedule extends React.Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+
+        const { league, leagueName, lookups } = this.props;
```

Comment:
leagueName shouldn't be getting passed through as a prop, calculate the league name with the league object (same as above, you have the helper method in LeagueHelpers)